packages/apollo-server-core/src/plugin/landingPage/default/index.ts

```diff
@@ -21,6 +21,7 @@ export interface ApolloServerPluginLandingPageDefaultBaseOptions {
   document?: string;
   variables?: Record<string, string>;
   headers?: Record<string, string>;
+  includeCookies?: boolean;
```

Comment:
Please update the doc comment above `document` to mention this (or better, split it up so each of the four options here have a doc comment since that's how doc comments work).

Ideally you should also add this to `LandingPageConfig` as well as to the two lines that look like

```
  const { version, __internal_apolloStudioEnv__, footer, ...rest } = options;
```
as explained in the comment above them.  Those lines should probably also have document, variables, and headers added, oops... I guess we're kind of in the "haven't quite updated the plugin yet" state for them.  The model here is a bit odd but basically works.

Please also update the doc website in `docs/source/api/plugin/landing-pages.md`. There's a README in `docs` explaining how to run the docs site locally; you can also see a Netlify deployment preview on the PR checks section.